### PR TITLE
Add tests for distribution strategy casp model

### PR DESCRIPTION
**Pull request by DavidKang**

### CODE REVIEW

File: src/api/spec/models/obs_factory/distribution_strategy_casp_spec.rb [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+      </resultlist>)
+    end
+  end
+
+  describe '#staging_manager' do
+    it { expect(strategy.staging_manager).to eq('caasp-staging-managers') }
+  end
+
+  describe '#repo_url' do
+    it { expect(strategy.repo_url).to eq('http://download.opensuse.org/distribution/13.2/repo/oss/media.1/build') }
```
Comment (coolo) [inline src/api/spec/models/obs_factory/distribution_strategy_casp_spec.rb]: I wouldn't say I expect that - the repo_url should be nil and the dashboard wouldn't show any repo for casp

File: src/api/spec/models/obs_factory/distribution_strategy_casp_spec.rb [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+  end
+
+  describe 'openqa_version' do
+    it { expect(strategy.openqa_version).to eq('1.0') }
```
Comment (coolo) [inline src/api/spec/models/obs_factory/distribution_strategy_casp_spec.rb]: wouldn't expect this either - you checked casp2

Comment (coolo) [reply src/api/spec/models/obs_factory/distribution_strategy_casp_spec.rb]: but as casp2 is no longer in development, it's hard to say. casp3's openqa_version is 3.0, so I wonder why the strategy returned 1.0 for 2

Comment (DavidKang) [reply src/api/spec/models/obs_factory/distribution_strategy_casp_spec.rb]: @coolo. openqa_version is hardcode with `1.0`

File: src/api/spec/models/obs_factory/distribution_strategy_casp_spec.rb (hunk 1 above)
Comment (DavidKang) [reply src/api/spec/models/obs_factory/distribution_strategy_casp_spec.rb]: @coolo, `repo_url` is hardcoded

File: src/api/spec/models/obs_factory/distribution_strategy_casp_spec.rb (hunk 2 above)
Comment (coolo) [reply src/api/spec/models/obs_factory/distribution_strategy_casp_spec.rb]: Very possible. I'm just saying I wouldn't *expect* current bugs - either fix them or leave them untested

Comment (DavidKang) [reply src/api/spec/models/obs_factory/distribution_strategy_casp_spec.rb]: So the current `openqa_version` for CASP should be `3.0`, right?

**Pull request merged**